WcaOnRails/config/locales/en.yml

```diff
@@ -365,6 +365,18 @@ en:
         guests_enabled:
           "true": "Ask about guests"
           "false": "Do NOT ask about guests"
+  mail_form:
```

Comment:
Oh I totally missed it!

Thanks, that's now clear!

Glad that we have our translation status page though, these could easily get forgotten :p